src/io.rs

```diff
@@ -256,5 +256,5 @@ where
         .from_reader(reader);
     csv_reader
         .into_deserialize()
-        .map(|e| e.map_err(|e| format_err!("err {}", e)))
+        .map(|e| e.map_err(|e| anyhow!("err {}", e)))
```

Comment:
I'm wondering, maybe we can take the opportunity to improve this piece of code. I believe you can entirely remove this `.map(...)` and replace the return value in the signature of the function with `impl Iterator<Item = std::result::Result<T, csv::Error>>`.

Reply:
That seems more relevant indeed.

Another solution would be to still perform the conversion but not forcing it into a string with something like that:
```suggestion
        .map(|e| e.map_err(Into::into))
```
This conversion will be performed by the `?` operator [later](https://github.com/CanalTP/navitia-poi-model/blob/c0880fbf0664a10c28f4598a8c31c0e6ffe03814/src/io.rs#L79) anyway and this will avoid to use `std::result::Result` were the whole module uses `anyhow::Result`. What would you think about that?

Reply:
The `Into::into` solution would be better. My primary concern was to introduce an additional step in your iteration (a `.map(...)` operation to apply on each item, an allocation for a `std::iter::Map` object) and they seem unnecessary (but I agree, performance might not be a concern here). At the very least, can you please do the `Into::into` modification?

Reply:
That's done :+1: